src/App.tsx

```diff
@@ -1,27 +1,48 @@
-import React from 'react';
 import './App.scss';
+import React, { useState } from 'react';
+import { Good } from './types/Good';
 import { GoodsList } from './GoodsList';
+import { getAll, get5First, getRedGoods } from './api/goods';
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState<Good[]>([]);
 
-export const App: React.FC = () => (
-  <div className="App">
-    <h1>Dynamic list of Goods</h1>
+  const handleClick = async (isLoading: () => Promise<Good[]>) => {
```

Comment:
Bad name for function argument.
```suggestion
  const handleClick = async (loaderCallback: () => Promise<Good[]>) => {
```